packages/plugins/i18n/server/src/services/ai-localizations.ts

```diff
@@ -1,6 +1,10 @@
-import type { Core } from '@strapi/types';
+import type { Core, Modules, UID } from '@strapi/types';
+import { traverseEntity } from '@strapi/utils';
+import { getService } from '../utils';
 
 const createAILocalizationsService = ({ strapi }: { strapi: Core.Strapi }) => {
+  const aiServerUrl = process.env.STRAPI_AI_URL || 'https://strapi-ai.apps.strapi.io';
```

Comment:
I think we should probably have a helper function for this somewhere so all ai features can use it